src/components/select/select.js

```diff
@@ -259,15 +273,39 @@ function SelectDirective($mdSelect, $mdUtil, $mdConstant, $mdTheming, $mdAria, $
     }
 
     var isMultiple = $mdUtil.parseAttributeBoolean(attr.multiple);
-
+       
+    //Set the separation options in a list
+    var selectionSeparatorCharOptions = { 
+      'comma' : ',',
+      'semicolon' : ';',
+      'pipe' : '|',
+      'none' : ''
+    };
+    var selectionSeparatorChar = selectionSeparatorCharOptions[attr.selectionSeparatorChar || 'comma'];
+    //Set the different spacing options with the selectionSeparatorChar
+    var selectionSeparator = ''; 
+    switch(attr.selectionSeparatorSpacing) {
+        case 'before':
+            selectionSeparator = ' '+selectionSeparatorChar;
+            break;
+        case 'none':
+            selectionSeparator = selectionSeparatorChar;
+            break;
+        case 'break':
+            selectionSeparator = selectionSeparatorChar+'<br>';
+            break;
+        default:
+            selectionSeparator = selectionSeparatorChar+' ';
+    }
+   
     // Use everything that's left inside element.contents() as the contents of the menu
     var multipleContent = isMultiple ? 'multiple' : '';
     var selectTemplate = '' +
       '<div class="md-select-menu-container" aria-hidden="true" role="presentation">' +
-      '<md-select-menu role="presentation" {0}>{1}</md-select-menu>' +
+      '<md-select-menu role="presentation" {0} selection-separator="{1}">{2}</md-select-menu>' +
```

Comment:
I would rather not have to apply this again here. Was `$attrs.selectionSeparator` not defined on https://github.com/angular/material/pull/11385/files#diff-d7b463f733c2e3fbc783f38073561b74R719 without this?